### DISCUSSION TITLE
Why do radio names change occasionally from the entered name?

### DISCUSSION THREAD

**Discussion by mikey60** [music-assistant/support#1779] [Q&A]
The names of radio stations randomly change from the value that I entered.  The URL remains the same.  I then have to go delete and re-enter the URL and name so my automations and scripts continue to function.

**Comment by OzGav** [comment]
What is it changing to? Does it always change to the same thing?

**Reply by mikey60** [reply]
I am not sure, but I believe that the name changes to the default name of the radio station. I like to use simple names end like "Classical Radio". This works better with a voice command.

**Reply by OzGav** [reply]
I’ll guess it resets after a MA restart. Can you try that?

**Reply by mikey60** [reply]
I restarted the MA Add-on version 2.0.0b81 and the Radio station names did not change.

**Reply by OzGav** [reply]
Interesting….if you can try and work out what causes it that would be great

**Reply by mikey60** [reply]
I will dig into the cause.
